README.md

```diff
@@ -23,60 +23,77 @@ Additional functionality may be built out in the future.
-   cd registry-client
-    ```
-   
-
-2. Create a virtual environment
-    ```
-    python -m venv venv
-    source ./venv/bin/activate
+    # Example assumes bash command shell. For others, consult shell documentation.
+    mkdir -p $HOME/.virtualenvs
+    python3 -m venv $HOME/.virtualenvs/registry-client
+    source $HOME/.virtualenvs/registry-client/bin/activate
     ```
 
-3. Install the tool to the virtual environment
+2. Install the tool to the virtual environment
     ```
-    pip install .
+    pip install pds.registry-client
     ```
 
-4. Run the tool directly
+3. Run the tool directly
     ```
-    ./cli/makesignedrequest --help
+    registry-client --help
     ```
+### Windows Installation
+For help installing Python, [see this documentation from PDS Deep Archive](https://nasa-pds.github.io/deep-archive/installation/index.html#installing-python-for-windows).
 
----
+To install the Registry Client software on Windows comprises the following steps:
 
-## BEGIN BOILERPLATE
+Installing Python 3.11 for Windows
+Creating a “virtual environment” to contain an isolated instance of Python 3.11
+Installing LXML 4.9.0 for Python 3.11 into the virtual environment
+Installing the PDS Deep Archive into the virtual environment
 
-[//]: # (TODO: Package as a proper CLI executable)
-If possible, make it so that your program works out of the box without any additional configuration—but see the [Configuration](###configuration) section for details.
+## Upgrading the Software
+To check and install an upgrade to the software, run the following command in your virtual environment (on Unix and Unix-like systems):
 
-To execute, run:
+```
+source $HOME/.virtualenvs/registry-client/bin/activate
+pip install --upgrade pds.registry-client
+```
 
-    (put your run commands here)
+Or on Windows in PowerShell:
 
-[//]: # (TODO: POPULATE THESE TEAMS)
-## 🏃 Getting Started With This Template 
+```
+.\pds\Scripts\activate.ps1
+pip install --upgrade pds.deeparchive
+```
+---
 
-See our wiki page for more info on setting up your new repo. You can remove this section once you have completed the necessary start-up steps.
 
-https://github.com/NASA-PDS/nasa-pds.github.io/wiki/Git-and-Github-Guide#creating-a-new-repo
+## Developer Quickstart
 
-**👉 Important!** You must assign the teams as mentioned on the wiki page above! At a minimum, these are:
+1. Clone the repository
+    ```
+   git clone https://github.com/NASA-PDS/registry-client.git
+   cd registry-client
+    ```
 
-| Team                                | Permission |
-| ----------------------------------- | ---------- |
-| `@NASA-PDS/pds-software-committers` | `write`    |
-| `@NASA-PDS/pds-software-pmc`        | `admin`    |
-| `@NASA-PDS/pds-operations`          | `admin`    |
 
----
+2. Create a virtual environment
+    ```
+    python -m venv venv
+    source ./venv/bin/activate
+    ```
 
+3. Install the tool to the virtual environment
+    ```
+    pip install .
```

Comment:
Should probably be `pip install --editable .[dev]`

Reply:
Fixed